Update sample.html

rawinslow24 opened the pull request:
I used new Date() to instantiate July 4, 1776. Then I used .getDay() to get the day of week for said date. Using the already existing HTML document, I switched out the Script for "Hello World" and placed the new script so that it returns the day in which July 4, 1776 took place.

rawinslow24 committed:
Update sample.html

I used new Date() to instantiate July 4, 1776. Then I used .getDay() to get the day of week for said date. Using the already existing HTML document, I switched out the Script for "Hello World" and placed the new script so that it returns the day in which July 4, 1776 took place.

Kilavagora commented:
@cp-winslow 

- We will have a separate topic on coding style, but in JavaScript convention is to use camelCase for variable names. E.g. indDay, instead of indday. 
- Keep indentation the same for the statements on the same level. 
- Also make you changes in separate file e.g. homework.html and keep sample.html as it was. That would avoid merge conflicts if I happened to update class 1 files

Other than that the code is correct :+1:

rawinslow24 committed:
#2 Update sample.html

I reverted back sample.html and created a new file (homework.html) for the assignment. Per the suggestions offerred, I updated the variables to meet camelCase and matched indentations (hopefully)
